app/src/main/java/br/org/cesar/knot_setup_app/view/scan/ScanFragment.java

```diff
@@ -173,9 +173,11 @@ public void onBluetoothPermissionRequired() {
         LogWrapper.Log("callbackOnBluetoothPermissionRequired", Log.DEBUG);
         Intent intent = new Intent(BluetoothAdapter.ACTION_REQUEST_ENABLE);
         this.startActivity(intent);
-        ActivityCompat.requestPermissions(getActivity(),
-                new String[]{Manifest.permission.ACCESS_FINE_LOCATION},
-                Constants.BLUETOOTH_PERMISSION_ID);
+        if(getActivity() != null){
```

Comment:
What will happen when getActivity is null? The permission wouldn't show and the user would not know what happened?

Reply:
Exactly, it was pointed out to me that when we can't handle some situation the proper response in android is to do nothing. I believe I could send the user some feedback (a toast maybe), but I am not sure whether that is an appropriate resolution.

Reply:
Please add this information on commit message.